init.lua

```diff
@@ -142,6 +142,7 @@ minetest.after(initialization_delay, _delayed_init)
 minetest.register_node(minetest.get_current_modname() .. ":craftdb", {
   description = "Digiline Minetest Recipe Database",
   groups = {cracky = 3, oddly_breakable_by_hand = 2},
+	is_ground_content = false,
```

Comment:
Please correct the indentation.  The existing code uses two-spaces and not tabs.